sdk/scheduler/src/main/java/org/apache/mesos/scheduler/plan/Element.java

```diff
@@ -92,8 +92,15 @@ default boolean isPending() {
     /**
      * Indicates whether the Element is in progress.
      */
-    default boolean isInProgress() {
-        return getStatus().equals(Status.IN_PROGRESS);
+    default boolean isPrepared() {
```

Comment:
I added a some documentation in the Status.java file, where all the states are defined.